src/components/custom/_signup-new.js

```diff
@@ -194,6 +194,19 @@ const StyledBinaryLogo = styled.img`
         width: 40px;
     }
 `
+const FooterNote = styled(Header)`
```

Comment:
You can pass type='small' prop for FooterNote. That way you wouldn't need to specify line-height and font size. The other properties can be passed as props as well

Reply:
Done.. Thanks for the suggestion @suthesan-deriv